app/template/assets/cui-2.0.0/js/vkaci.js

```diff
@@ -124,7 +124,7 @@ function neo_viz_config(showPodName, container, cypher, seed = null) {
     };
 
     if (seed) {
-        config.layout.randomSeed = seed
+        config.layout.randomSeed = parseFloat(seed)

```

Comment:
I kept the randomseed as a string because that’s the seed that the neovis lib returns when you ask for it. The error being produced on the frontend is because of inconsistent variable types within the lib. When you parse in the seed into a float half of it gets removed and you may end up with an overlapping visualization for example the edges crossing each other. I remember working on this really extensively last year since we wanted to avoid this issue.